velox/exec/Exchange.h

```diff
@@ -312,7 +338,13 @@ class Exchange : public SourceOperator {
             exchangeNode->id(),
             "Exchange"),
         planNodeId_(exchangeNode->id()),
-        exchangeClient_(std::move(exchangeClient)) {}
+        exchangeClient_(std::move(exchangeClient)) {
+    // ExchangeClient is shared among multiple drivers' Exchange operators so we
+    // only need one of the operator to set it up.
+    if (exchangeClient_->pool() == nullptr) {
+      exchangeClient_->setMemoryPool(operatorCtx_->pool());
```

Comment:
Can we set the memory pool when creating exchange client using task's memory pool in place such as Task::start?